docs/api/ref/schemas/image.yaml

```diff
@@ -1,34 +1,40 @@
-type: object
-title: image
-description: |
-  This object represent an image that was uploaded to a product.
-  "imgid" is an integer which is a sequential number unique to each picture.
-properties:
-  sizes:
-    type: object
-    description: |
-      The available image sizes for the product (both reduced and full). 
-      The reduced images are the ones with numbers as the key( 100, 200 etc) 
-      while the full images have `full` as the key.
-    properties:
-      full:
-        $ref: ./image_size.yaml
-    patternProperties:
-      "(?<image_size>100|400)":
-        description: |
-          properties of thumbnail of size `image_size`.
-          **TODO** explain how to compute name
+components:
+  schemas:
+    Sizes:
+      type: object
+      description: |
+        The available image sizes for the product (both reduced and full). 
+        The reduced images are the ones with numbers as the key( 100, 200 etc) 
+        while the full images have `full` as the key.
+      properties:
+        full:
+          $ref: ./image_size.yaml
+      patternProperties:
+        "(?<image_size>100|400)":
+          description: |
+            properties of thumbnail of size `image_size`.
+            **TODO** explain how to compute name
 
-          For real type: see description of property `full`.
-          (Put this way because of a [bug in rapidoc](https://github.com/rapi-doc/RapiDoc/issues/880))
-        type: string
-  uploaded_t:
-    type: string
-    example: "1457680652"
-    description: |
-      The time the image was uploaded (as unix timestamp).
-  uploader:
-    type: string
-    example: openfoodfacts-contributors
-    description: |
-      The contributor that uploaded the image.
+            For real type: see description of property `full`.
+            (Put this way because of a [bug in rapidoc](https://github.com/rapi-doc/RapiDoc/issues/880))
+          type: string
+
+    Image:
+      type: object
+      title: image
+      description: |
+        This object represent an image that was uploaded to a product.
+        "imgid" is an integer which is a sequential number unique to each picture.
+      properties:
+        sizes:
+          $ref: "#/components/schemas/Sizes"
+        uploaded_t:
+          type: string
```

Comment:
all *_t timestamps should be integers in fact

Reply:
![image](https://github.com/user-attachments/assets/9cbcbd9b-4f44-473e-a8d4-26d5784cc8b6)

You're right! But they are currently returned as "false" strings... Is there a change planned in the immediate future? If we put it in multi-type the generator currently gets stuck.

Reply:
they were integer in the products I tested, but it's something we have a lot of issues with, there's no type in Perl, so the JSON output can have integers converted to strings depending on how they were used last